### PR TITLE
kubevirt_vm: simplify previous logic change

### PR DESCRIPTION
**Pull request by mmazur**
##### SUMMARY
Previous patch could've been done simpler, hence this one.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
modules/kubevirt_vm

### CODE REVIEW

**Comment by mmazur** [comment]
It's not equivalent…

**Comment by mmazur** [comment]
Strike that, they are. Uni–level logic to the rescue, I knew those years of uni would come in handy one day. :)

File: lib/ansible/modules/cloud/kubevirt/kubevirt_vm.py [hunk 1]
```diff
@@ -414,7 +414,7 @@ def execute_module(self):
         # Changes in VM's spec or any changes to VMIs warrant a full CRUD, the latter because
         # VMIs don't really have states to manage; they're either present or don't exist
         # Also check_mode always warrants a CRUD, as that'll produce a sane result
-        if vm_spec_change or (ephemeral and vm_spec_change) or k8s_state == 'absent' or self.check_mode:
+        if vm_spec_change or k8s_state == 'absent' or self.check_mode:
```
Comment (machacekondra) [inline lib/ansible/modules/cloud/kubevirt/kubevirt_vm.py:419]: Can you please change the comment so it better describe the if

Comment (mmazur) [reply lib/ansible/modules/cloud/kubevirt/kubevirt_vm.py:419]: Good point. Done.

**Comment by machacekondra** [comment]
shipit

**Pull request merged**